### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by XhmikosR**
/CC @cvrebert

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,13 +53,13 @@ All options are passed directly to css-flip's `flip()` function.
 None of the options are required.
 
 #### options.compress
-Type: `Boolean`
+Type: `Boolean`
```
Comment (cvrebert) [inline README.md]: Remove trailing whitespace

Comment (XhmikosR) [inline README.md]: It's done on purpose to enforce a newline.

Comment (XhmikosR) [inline README.md]: I switched to a `ul`, please check again.

**Comment by cvrebert** [comment]
Why support so many Node.js versions? E.g. BS4 only supports 4 & 5.

**Comment by XhmikosR** [comment]
It's your call, I just went for what you had in package.json.

PS. I added one more patch to this branch.

**Comment by XhmikosR** [comment]
Perhaps we should get rid of jahint and jscs and just make ESLint's config stricter.

File: package.json [hunk 2]
```diff
@@ -20,19 +20,19 @@
...
     "grunt-contrib-nodeunit": "^0.4.1",
-    "grunt-eslint": "^10.0.0",
-    "grunt-jscs": "^1.5.0",
+    "grunt-eslint": "^18.0.0",
+    "grunt-jscs": "^2.7.0",
     "load-grunt-tasks": "^3.1.0"
   },
   "peerDependencies": {
-    "grunt": "~0.4.3"
+    "grunt": ">=0.4.5"
```
Comment (cvrebert) [inline package.json]: `>=` seems dangerously wide

File: README.md (hunk 1 above)
Comment (cvrebert) [inline README.md]: Good call

File: package.json (hunk 2 above)
Comment (XhmikosR) [inline package.json]: Well, http://gruntjs.com/blog/2016-02-11-grunt-1.0.0-rc1-released#peer-dependencies
https://twitter.com/gruntjs/status/700819604155707392

**Comment by XhmikosR** [comment]
I made https://github.com/twbs/grunt-css-flip/tree/lint, feel free to comment and merge after this one is merged.

**Comment by XhmikosR** [comment]
Updated.

File: package.json (hunk 2 above)
Comment (cvrebert) [inline package.json]: OK

**Pull request merged**